src/ee/store/_ee_store_sqlite.py

```diff
@@ -57,8 +57,14 @@ def create(cls, db=""):
             conn_str = f"sqlite:///{db}"
         else:
             conn_str = "sqlite://"  # in memory
+        conn_str = f"{conn_str}?check_same_thread=False"
         engine = create_engine(conn_str, echo=EE_DEBUG)
-        Base.metadata.create_all(engine)
+        try:
+            Base.metadata.create_all(engine)
+        except OperationalError:
+            # already created?
+            # TODO: do this properly
```

Comment:
do this properly

Reply:
where is the issues related to this implementation?

Reply:
Issue #15